components/publish/SubmitTestimonyForm.tsx

```diff
@@ -33,10 +34,13 @@ export const SubmitTestimonyForm = () => {
             <Form step={form.step} bill={form.bill} synced={form.synced} />
           </Col>
           <Col xs={3}>
-            {["publish", "share"].includes(form.step) ? (
-              <PublishInfo />
-            ) : (
-              <QuickInfo bill={form.bill} profile={form.profile} />
+            {form.step == "position" ? (
+              ["publish", "share"].includes(form.step) ? (
+                <PublishInfo />
```

Comment:
If `form.step =="position"`, then wouldn't `["publish", "share"].includes(form.step)`, always be false?

Reply:
Good point! I guess I don't need the `<PublishInfo />` anymore, and I could simplify the nested if statements.